aries-site/src/examples/templates/forms/formHelpers.js

```diff
@@ -15,6 +15,58 @@ export const emailMask = [
   },
 ];
 
+export const dateMask = [
+  {
+    length: [1, 2],
+    options: Array.from({ length: 12 }, (v, k) => k + 1),
+    regexp: /^1[0,1-2]$|^0?[1-9]$|^0$/,
+    placeholder: 'MM',
+  },
+  { fixed: '/' },
+  {
+    length:  2,
+    options: Array.from({ length: 6 }, (v, k) => 20 + k),
+    regexp: /^2[0,1-5]$|^2?$/,
```

Comment:
Saw your comment about restricting dates. I like the forward thinking and inspiration. In this case, I would recommend we think of the `mask` as specifying the "types" of values, but not restrict the actual values being entered. For the actual validation of the input, I would use a validation rule (though that might be overkill for this example case -- but could be fun to implement).

Recommendation:
- Remove the 20-25 restriction on the mask
- Play with a validation rule which is a function which compares the value and ensures the value is >= to today's date, but less than today's date plus years (in a real world implementation we'd likely lean on whatever payment service's API, but supplement it with some guardrails aiming to limit any erroneous data prior to the fetch call to the API)